### ISSUE TITLE
Update password link

### ISSUE THREAD

**Issue by dernettemann118** [Microsoft/adfsWebCustomization#39]
Hello @all,

how can we configure the passwordchange page to be displayed not in the footer and not in dark display color?

It would be nice to see the change password option in the white centered area.

**Comment by bongiovimatthew-microsoft** [comment]
Thanks @dernettemann118. How is the password link in the footer set up in your environment (meaning what were the ADFS PowerShell cmdlets you ran to put it in place)? 

To add a password link to the main body of the page, you can use the [sign-in page description](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/add-sign-in-page-description), which supports HTML href links to your password reset page.

**Comment by dernettemann118** [comment]
@bongiovimatthew-microsoft 
Thanks for your input. The password link is in ADFS the "helpdesk link". I will try the sign-in page with HTML - really thank you for that suggestion. Sometimes it is to easy to see it by myself ;)

**Comment by dernettemann118** [comment]
Tested already yesterday, works. But for now it isn't so nice integrated like the links azure ad sign xp ...

![untitled](https://user-images.githubusercontent.com/39616319/42075511-8a6a6178-7b71-11e8-836f-70d45f761063.png)

**Comment by bongiovimatthew-microsoft** [comment]
Yeah, I see what you mean. I'm going to update the title of this issue, and mark it as help wanted in case anyone is able to play around with the CSS to get this to look better.

**Comment by dernettemann118** [comment]
Same here:
https://github.com/Microsoft/adfsWebCustomization/issues/35

If I insert the suggested code with enabled KMSI it doesnt look really beautiful
![adfs](https://user-images.githubusercontent.com/39616319/42147181-cfc47256-7dcc-11e8-92e3-7fe0a3f17b0f.png)

**Comment by brentcarlson** [comment]
I also used the sign-in page description setting to solve for this. This is the same issue that also applies to sign in options looking misaligned like show in this thread. As I mentioned in issue #38 I was able make this look a little better by adding "padding-button: 25px;" to the #loginForm element in the CSS, but its not ideal as the Auth Options still get shifted too high when the Back button is used. However, its much better. The same solution makes both issues better (Sign in options and sign-in description). Here a screenshot:

![image](https://user-images.githubusercontent.com/32167232/42228088-af7aa204-7ea8-11e8-9fab-05541239a5cd.png)
